examples/gameserve-windows.yaml

```diff
@@ -0,0 +1,103 @@
+    # port is available. When static is the policy specified, `hostPort` is required to be populated
+    # - "Passthrough" dynamically sets the `containerPort` to the same value as the dynamically selected hostPort.
+    #      This will mean that users will need to lookup what port has been opened through the server side SDK.
+    portPolicy: Static
+    # The name of the container to open the port on. Defaults to the game server container if omitted or empty.
+    container: simple-game-server
+    # the port that is being opened on the game server process
+    containerPort: 7654
+    # the port exposed on the host, only required when `portPolicy` is "Static". Overwritten when portPolicy is "Dynamic".
+    hostPort: 7777
+    # protocol being used. Defaults to UDP. TCP and TCPUDP are other options
+    protocol: UDP
+  # Health checking for the running game server
+  health:
+    # Disable health checking. defaults to false, but can be set to true
+    disabled: false
+    # Number of seconds after the container has started before health check is initiated. Defaults to 5 seconds
+    initialDelaySeconds: 5
+    # If the `Health()` function doesn't get called at least once every period (seconds), then
+    # the game server is not healthy. Defaults to 5
+    periodSeconds: 5
+    # Minimum consecutive failures for the health probe to be considered failed after having succeeded.
+    # Defaults to 3. Minimum value is 1
+    failureThreshold: 3
+  # Parameters for game server sidecar
+  sdkServer:
+    # sdkServer log level parameter has three options:
+    #  - "Info" (default) The SDK server will output all messages except for debug messages
+    #  - "Debug" The SDK server will output all messages including debug messages
+    #  - "Error" The SDK server will only output error messages
+    logLevel: Info
+    # grpcPort and httpPort control what ports the sdkserver listens on.
+    # Starting with Agones 1.2 the default grpcPort is 9357 and the default
+    # httpPort is 9358. In earlier releases, the defaults were 59357 and 59358
+    # respectively but as these were in the ephemeral port range they could
+    # conflict with other TCP connections.
+    grpcPort: 9357
+    httpPort: 9358
+  # [Stage:Alpha]
+  # [FeatureFlag:PlayerTracking]
+  # Players provides the configuration for player tracking features.
+  # Commented out since Alpha, and disabled by default
+  # players:
+  #   # set this GameServer's initial player capacity
+  #   initialCapacity: 10
+  #
+  # Pod template configuration
+  template:
+    # pod metadata. Name & Namespace is overwritten
+    metadata:
+      labels:
+        myspeciallabel: myspecialvalue
+    # Pod Specification
+    spec:
+      containers:
+      - name: simple-game-server
+        image:  gcr.io/agones-images/simple-game-server:0.2
+        imagePullPolicy: Always
+      # Limit this pod to Windows nodes.
+      nodeSelector:
```

Comment:
Is this the only difference between this file and https://github.com/googleforgames/agones/blob/main/examples/gameserver.yaml?

So that we don't have to maintain two nearly identical files (as we add / graduate features, update docs, etc) what about adding this part commented out into the existing file, and having the instructions say that it needs to be uncommented _if you are running a cluster with both windows and linux game server nodes_. If I understand correctly, you could use the existing file _without_ the node selector stanza if you created a cluster where you had a linux node pool for the agones controller (with the node taints and labels) and the rest of the cluster was windows node pools.

Reply:
Yeah, I would +1 that. Since the gameserver.yaml shows all the options 👍🏻